### PR TITLE
Prevent non array attachments

### LINKED ISSUE #2389: Blank channel when sending attachments as object
RocketChat v0.19.0
Running as a Docker container using compose on Debian amd64 Jessie.

Hi all,
I have integrated a Zapier webhook into RocketChat to get new tweets from my twitter list and send them to a `news` channel. Firstly it worked fine, then I started to play around with the format so that it matched what RocketChat was expecting. So using the double underscore approach to produce the nested data I finished the Zap and began running it. So now I get the messages within the channel, but the whole channel is blank. Using the inspector shows the `<div></div>` which is supposed to contain my messages is empty.

rocket-chat_blank-channel

For the sake of trying to understand whats going on here, this is what I get from `docker logs <container_id>, so I know that it's not Zapier as it's sending the data correctly.`:

```javascript
@bodyParams { 
  text: 'Cabinet Office',
  attachments: 
  {
    color: '#2196F3',
    image_url: 'http://pbs.twimg.com/profile_images/664783281045532672/WZFsBWMR_normal.png',
    title_url: 'https://twitter.com/cabinetofficeuk/status/704618458852548608',
    title: 'The Independent Commission on Freedom of Information final report is now published https://t.co/N0nV5gmSmR' 
  } 
}
```

I really love RocketChat :rocket: guys so keep up the good work. Hopefully this is something trivial :smile: 

Thanks in advance,
K

### PR DESCRIPTION
**Pull request by rodrigok**
Closes #2389

### CODE REVIEW

File: packages/rocketchat-integrations/server/processWebhookMessage.js [hunk 1]
```diff
@@ -67,6 +68,11 @@ this.processWebhookMessage = function(messageObj, user, defaultValues) {
 			throw new Meteor.Error('invalid-channel-type');
 	}
 
+	if (messageObj.attachments && !_.isArray(messageObj.attachments)) {
+		console.log('Attachments should be Array, ingnoring value'.red, messageObj.attachments);
```
Comment (marceloschmidt) [inline packages/rocketchat-integrations/server/processWebhookMessage.js]: typo: ingnoring -> ignoring

File: packages/rocketchat-integrations/server/processWebhookMessage.js [hunk 2]
```diff
@@ -53,13 +53,14 @@ this.processWebhookMessage = function(messageObj, user, defaultValues) {
...
 			});
-			if (roomUser == null && room == null) {
+			if (!_.isObject(roomUser) && !_.isObject(room)) {
 				throw new Meteor.Error('invalid-channel');
 			}
 			if (!room) {
 				Meteor.runAsUser(user._id, function() {
 					Meteor.call('createDirectMessage', roomUser.username);
-					return room = RocketChat.models.Rooms.findOne(rid);
+					room = RocketChat.models.Rooms.findOne(rid);
```
Comment (rodrigok) [inline packages/rocketchat-integrations/server/processWebhookMessage.js]: Problem resolved

**Comment by marceloschmidt** [comment]
lgtm

**Comment by rodrigok** [comment]
lgtm

**Comment by engelgabriel** [comment]
lgtm

**Pull request merged**